README.md

```diff
@@ -56,11 +60,6 @@ Want to see some existing examples? Check out:
 
 I'd love PRs adding more.
 
-Notes:
-
-* The `Dangerfile.js` needs to be able to run on node without running through babel right now.
-* The shape of the API is [`git`](https://github.com/danger/danger-js/blob/master/source/dsl/git.js) and [`pr`](https://raw.githubusercontent.com/danger/danger/master/spec/fixtures/github_api/pr_response.json)
```

Comment:
The `git` link is outdated, and the API shape is noted further up in the README, so I thought this could be removed.